### PR TITLE
fix(plugin-image): display src field idea

### PR DESCRIPTION
**Pull request by hejtful**
Fix proposal for https://github.com/serlo/frontend/issues/2646

With this fix, 16 tests pass and 2 fail.

### CODE REVIEW

**Comment by elbotho** [comment]
> because we currently display image toolbar inside of image plugin, even when it's a child of multimedia. (is this a bug or intended behavior?)

Yes this is intended. We could create logic that combines multimedia + media-child toolbar buttons but then we will run into problems with space I'm afraid (image toolbar will grow in the future :) )

File: src/serlo-editor/plugins/image/renderer.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export function ImageRenderer({
         {wrapWithLink(
           placeholder ?? (
             <img
-              className="serlo-img"
+              className="serlo-img serlo-img-error-placeholder"
```
Comment (Entkenntnis) [inline src/serlo-editor/plugins/image/renderer.tsx]: you can add tailwind directly here.

File: src/assets-webkit/styles/serlo-tailwind.css [hunk 2]
```diff
@@ -115,6 +115,17 @@
...
+    @apply relative min-h-[100px] min-w-[100%];
+  }
+  .serlo-img-error-placeholder::before {
+    @apply absolute left-0 top-[50%] h-full w-full translate-y-[-50%] border-2 border-editor-primary-100 bg-slate-100;
+    content: '';
+  }
+  .serlo-img-error-placeholder::after {
+    @apply absolute left-0 top-1 top-1 block h-full w-full;
+    content: '\2639'' ' attr(alt);
+  }
```
Comment (Entkenntnis) [inline src/assets-webkit/styles/serlo-tailwind.css]: the styles here are mainly for components that are used in multiple places and need overriding attributes. For one time usage you can directly use tailwind.

**Review by Entkenntnis — COMMENTED** [review]
small hint

File: src/serlo-editor/plugins/image/renderer.tsx (hunk 1 above)
Comment (elbotho) [reply src/serlo-editor/plugins/image/renderer.tsx]: @hejtful I'm pretty strict about adding editor stuff into the renderer. The renderer should remain as "pure" as possible.
What we need in the long term is a smarter Image component that parses the url, tries to load the image etc…
In the short term something like that would be okay, but maybe you could add a comment about that.

File: src/assets-webkit/styles/serlo-tailwind.css (hunk 2 above)
Comment (elbotho) [reply src/assets-webkit/styles/serlo-tailwind.css]: maybe relevant: https://tailwindcss.com/docs/content  :)

File: src/serlo-editor/plugins/image/renderer.tsx (hunk 1 above)
Comment (elbotho) [reply src/serlo-editor/plugins/image/renderer.tsx]: @hejtful adding it in this div here should also work maybe?
https://github.com/serlo/frontend/blob/c3fb04da40b1b16e6a61c41dc1ed69b1451626df/src/serlo-editor/plugins/image/editor.tsx#L57

Comment (hejtful) [reply src/serlo-editor/plugins/image/renderer.tsx]: > adding it in this div here should also work maybe?

I couldn't make this work, unfortunately. Added a comment, though.

**Review by Entkenntnis — APPROVED** [review]
tailwind looks good now

**Pull request merged**